src/main/resources/static/js/comment.js

```diff
@@ -96,7 +98,7 @@ const CommentApp = (function () {
             deleteComment();
         };
 
-        return { init };
+        return {init};
```

Comment:
```suggestion
        return { init };
```
별로 중요한 것은 아닌데, 자바스크립트 쪽에서의 관례는 보통 이렇다고 알고 있습니다.